orttraining/orttraining/test/python/orttraining_test_orttrainer_frontend.py

```diff
@@ -1499,25 +1515,44 @@ def _adam_max_norm_clip_data():
     device_capability_major = torch.cuda.get_device_capability()[0]
     if device_capability_major == 7:    # V100 for Dev machine
         return [
-            (0, 'cuda', 1.0, 1, 12, [10.592951, 10.067989, 9.619152, 9.245731, 8.881137,\
-                8.578644, 8.280573, 8.063023, 7.797933, 7.486215, 7.233806, 7.011791]),
-            (0, 'cuda', 0.1, 1, 12, [10.592951, 10.068722, 9.620503, 9.247791, 8.883972,\
-                8.582286, 8.285027, 8.068308, 7.803638, 7.492318, 7.240352, 7.018665]),
-            (42, 'cuda', 1.0, 1, 12, [10.647908, 10.144501, 9.672352, 9.306980, 8.956026,\
-                8.602655, 8.351079, 8.088144, 7.867220, 7.564082, 7.289846, 7.073726]),
-            (42, 'cuda', 0.1, 1, 12, [10.647908, 10.145191, 9.673690, 9.309031, 8.959020,\
-                8.606632, 8.355836, 8.093478, 7.873327, 7.570731, 7.296772, 7.0809422]),
+            (0, 'cuda', 1.0, 1, 12, {
+                12: [10.592951, 10.067989, 9.619152, 9.245731, 8.881137,
+                     8.578644, 8.280573, 8.063023, 7.797933, 7.486215, 7.233806, 7.011791],
+                14: [10.584141, 10.068119, 9.581743, 9.191472, 8.880169, 8.5352,
+                     8.311425,  8.061202, 7.773032, 7.523009,  7.258711, 7.02805]}),
+            (0, 'cuda', 0.1, 1, 12, {
+                12: [10.592951, 10.068722, 9.620503, 9.247791, 8.883972,
+                     8.582286, 8.285027, 8.068308, 7.803638, 7.492318, 7.240352, 7.018665],
+                14: [10.584141, 10.068845, 9.583107, 9.193537, 8.882966, 8.538839,
+                     8.315872,  8.066408, 7.778978, 7.529708, 7.265849, 7.035439]}),
+            (42, 'cuda', 1.0, 1, 12, {
+                12: [10.647908, 10.144501, 9.672352, 9.306980, 8.956026,
+                     8.602655, 8.351079, 8.088144, 7.867220, 7.564082, 7.289846, 7.073726],
+                14: [10.697515, 10.229034, 9.765422, 9.428294, 9.080612, 8.715208,
+                     8.459574, 8.169073, 7.940211, 7.654147, 7.390446, 7.166227]}),
```

Comment:
Are you missing `(42, 'cuda', 0.1, 1, 12...` here?

Reply:
Thanks, I missed it.